RoddyCore/src/de/dkfz/roddy/execution/jobs/Job.groovy

```diff
@@ -164,7 +164,7 @@ class Job extends BEJob<BEJob, BEJobResult> {
                 , []
                 , [:]
                 , Roddy.getJobManager()
-                , JobLog.toOneFile(context.loggingDirectory)
+                , JobLog.toOneFile(new File (context.loggingDirectory, jobName + ".o{JOB_ID}"))
```

Comment:
Still works for PBS? Just curious

Reply:
Indeed no. There is a corresponding PR in BE (https://github.com/eilslabs/BatchEuphoria/pull/86)